app/helpers/injection_helper.rb

```diff
@@ -12,6 +12,16 @@ def inject_enterprises(enterprises = Enterprise.activated.includes(address: :sta
     )
   end
 
+  def inject_groups
+    select_only = required_attributes EnterpriseGroup, Api::GroupListSerializer
+
+    inject_json_ams(
+      'groups',
+      EnterpriseGroup.on_front_page.by_position.select(select_only).includes(address: :state).all,
+      Api::GroupListSerializer
+    )
+  end
+
```

Comment:
OCD mode triggered... can you please make the double quotes (or single quotes) consistent in this file? :-)

Reply:
:-D